### PR TITLE
Style: Darken TOC in dark mode and move style to fw.css

### LINKED ISSUE #761: [Bug] Table of contents looks too bright in dark mode
**URL**
[See software manual(https://jamulus.io/wiki/Software-Manual)

**Describe the bug**
The table of contents background looks too bright in dark mode. Some designer should look at it.

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Darken TOC background in dark mode. This leads to the following:
toc

This also moves the css for details into fw.css as the style is side 
wide and not restricted only to the wiki.

**Context: Fixes an issue? Related issues**
Fixes: https://github.com/jamulussoftware/jamuluswebsite/issues/761

**Status of this Pull Request**
Ready for review. @mulyaj could you please have a look at the contrast?

**What is missing until this pull request can be merged?**
Ready for review

**Does this need translation?**

NO

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: assets/css/fw.css [hunk 1]
```diff
@@ -295,6 +295,22 @@ main>div {
   margin: 0 .3em 0 0;
 }
 
+details {
+  font: 16px;
+  font-style: bold;
+  width: 620px;
+}
+
+details>summary {
```
Comment (pljones) [inline assets/css/fw.css:304]: Shouldn't `>` have spaces around it?

Comment (ann0see) [reply assets/css/fw.css:304]: It doesn't need to, I think, but it can. Would you prefer that?

Comment (pljones) [reply assets/css/fw.css:304]: It did have and I don't see a reason to make the code less readable.  So yes.

(You added spaces elsewhere ;) )

Comment (ann0see) [reply assets/css/fw.css:304]: Ok. Fixed.

File: assets/css/fw.css [hunk 2]
```diff
@@ -398,11 +414,15 @@ main>div {
     background-color: #2b2f2f;
   }
 
-
   pre code {
     border-left-color: #3daed2;
   }
 
+  details>summary {
```
Comment (ann0see) [inline assets/css/fw.css:421]: no... not fully.

File: assets/css/fw.css (hunk 1 above)
Comment (ann0see) [reply assets/css/fw.css:304]: Hmm, no: The atom beautifier reverts it.

Comment (pljones) [reply assets/css/fw.css:304]: Oh well, if that's "correct" according to some idea of CSS styling, then fine.

**Review by pljones — APPROVED** [review]

**Pull request merged**